orchestra/todos/filters.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import filters
+
+
+class QueryParamsFilterBackend(filters.BaseFilterBackend):
```

Comment:
I think we're OK, but does this open us up to any injection attacks? If it does, what kind of sanitization should we be doing?

Reply:
```
Django’s querysets are protected from SQL injection since their queries are constructed using query parameterization. A query’s SQL code is defined separately from the query’s parameters. Since parameters may be user-provided and therefore unsafe, they are escaped by the underlying database driver.
Django also gives developers power to write raw queries or execute custom sql. These capabilities should be used sparingly and you should always be careful to properly escape any parameters that the user can control. In addition, you should exercise caution when using extra() and RawSQL.
```
https://docs.djangoproject.com/en/3.1/topics/security/#sql-injection-protection

According to the docs, `extra()` and `RawSQL` methods/functions are unsafe, but querysets are injection-proof.
In our case, we are not using `extra()` and `RawSQL`.

Besides, a given serializer only deals with the fields specified in its meta class, other fields are ignored (as well as nested fields, like 'todo__project__id', unless they are explicitly specified in the serializer's fields).

Here's a good article from Jacob Kaplan-Moss Jacob Kaplan-Moss: https://jacobian.org/2020/may/15/preventing-sqli/ (by the way, it contains a good idea for our upcoming hackathon).